aiida_common_workflows/workflows/relax/cp2k/protocol.yml

```diff
@@ -34,6 +34,7 @@ moderate:
         MAX_SCF:       20
         EPS_SCF:       1.0E-7
         MAX_ITER_LUMO: 10000
+        SCF_GUESS:     RESTART
```

Comment:
I would suggest deleting this line, as it is added automatically by the restart handler:
- see here: https://github.com/aiidateam/aiida-cp2k/blob/develop/aiida_cp2k/workchains/base.py#L89
- and here: https://github.com/aiidateam/aiida-cp2k/blob/0916da4d53ed056a84e25de5e8691841ee23bbab/aiida_cp2k/utils/input_generator.py#L179

Reply:
@yakutovicha Good catch, thanks! I have removed `SCF_GUESS`.